integration_tests/pages/apply/isPersonTransgender.ts

```diff
@@ -0,0 +1,17 @@
+import { ApprovedPremisesApplication } from '../../../server/@types/shared'
+import ApplyPage from './applyPage'
+
+export default class IsPersonTrangender extends ApplyPage {
+  constructor(application: ApprovedPremisesApplication) {
+    super(
+      `Is ${application.person.name} transgender or do they have a transgender history?`,
```

Comment:
Just wondering if we can automatically glean this from the `genderIdentity` field? For example if their `sex` is Female, but their `genderIdentity` is Male (or vice versa - not even sure how we'd deal with non-binary folks), then we know we need to ask the question. It might even the worth going to the dropout part from the off if we know the person is a cisgender woman, as we don't support female APs at the mo.

I think this is probably OK to ship as it is though, but I'll check in with the team about what the idea flow would be.

Reply:
Yeah this is a valid point. I would be wary of automating it though, I think we need to allow for some human input on it. It doesn't seem like much hassle to answer a one yes/no question. 
I think it is worth flagging early if the applicant if a cis woman to say we don't support female APs otherwise people could waste a fair amount of time.